frame/babe/src/lib.rs

```diff
@@ -147,11 +147,11 @@ decl_storage! {
 		/// We reset all segments and return to `0` at the beginning of every
 		/// epoch.
 		SegmentIndex build(|_| 0): u32;
-		UnderConstruction: map hasher(twox_64_concat) u32 => Vec<schnorrkel::RawVRFOutput>;
+		UnderConstruction: map hasher(twox_64_concat) u32 => Vec<schnorrkel::Randomness>;
```

Comment:
This should not require migration because under the hood the `Randomness` type is equal to the `RawVRFOutput` type?

Reply:
But the underlying values would be different right?

Reply:
Yes, `randomness = H( RawVRFOutput, H(transcript_stuff,babe_public_key) )`. Adding `transcript_stuff` makes us fit the VRF definition from the Praos paper.  Adding `babe_public_key` avoids validators missusing HDKD.

Reply:
The underlying values would be different if `UnderConstruction` would be non-empty. But in that case we've already kicked off the beacon incorrect so it should not alter our security assumption in that case.